### PR TITLE
Create User for Admin

### PR DESCRIPTION
**Pull request by JFronny**

### CODE REVIEW

File: src/main/java/edu/kit/hci/soli/controller/LoginControllerAdvice.java [hunk 1]
```diff
@@ -28,31 +29,35 @@ public CsrfToken getCsrfToken(HttpServletRequest request) {
...
-        String email = oidcUser.getUserInfo().getEmail();
-        User user = userService.findByEmail(email);
+        String id = "kit/" + principal.getName();
+        User user = userService.findByUserId(id);
         if (user == null) {
-            log.info("Creating new OIDC user: {} with email: {}", username, email);
-            user = new User();
-            user.setEmail(email);
+            log.info("Creating new OIDC user: {} with id: {}", username, id);
+            user = new edu.kit.hci.soli.domain.OidcUser();
```
Comment (Jan8192) [inline src/main/java/edu/kit/hci/soli/controller/LoginControllerAdvice.java:55]: Please import it. Is this for some reason intentional?

**Review by Jan8192 — CHANGES_REQUESTED** [review]
This looks, other than the one thing, good to me

File: src/main/java/edu/kit/hci/soli/controller/LoginControllerAdvice.java (hunk 1 above)
Comment (JFronny) [reply src/main/java/edu/kit/hci/soli/controller/LoginControllerAdvice.java:55]: We already import a class called `OidcUser` (used in the method declaration)

**Review by Jan8192 — APPROVED** [review]
Alright! This looks fine!

**Pull request merged**